vtex/faststore
# feat: Search suggestions term count

## What's the purpose of this pull request?
Adds suggestion term's popularity count to GraphQL API

## How it works? 
Currently, we can query for term suggestions and top searches. However, there's no way to know the occurrences of these terms. I propose changing `StoreSuggestions.term` from `string[]` to `Array<{value: string, count: number}>`. 

## How to test it?
Try the following query:
```
query Suggestions {
  search(first: 10, term: "") {
    suggestions {
      terms {
        value
        count
      }
    }
  }
}
```

You should get something like:
```
{
  "data": {
    "search": {
      "suggestions": {
        "terms": [
          {
            "value": "salad",
            "count": 473
          },
          {
            "value": "monitor",
            "count": 472
          },
          {
            "value": "cotton",
            "count": 467
          }
        ]
      }
    }
  }
}
```

### Starters Deploy Preview
- https://github.com/vtex-sites/nextjs.store/pull/73
- https://github.com/vtex-sites/gatsby.store/pull/71
